tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -230,6 +232,36 @@ func (ls *leaseStresser) createLeases() {
 	wg.Wait()
 }
 
+func (ls *leaseStresser) createShortTermLeases() {
+	neededLeases := ls.numLeases - len(ls.shortLivedLeases.getLeasesMap())
```

Comment:
for short team leases, we do ever revoke them in one test case? we probably should just create shortteam lease once per test case?